.gitignore

```diff
@@ -0,0 +1,9 @@
+# Files here are ignored by git
+
+node_modules/
+
+test.md
+
+.hintrc
+
+.stylelintrc.json
```

Comment:
### Optional suggestions

- [ ] You don't need to include the following files inside the .gitignore

- .hintrc
- .stylelintrc.json

Reply:
I do not think this is enough reason to stop my approval. Those files handle the linters for my local environment not the linters on Github. My .github folder contains all necessary subfolders and files.

Reply:
The issue is, you added the files where it shouldn't be. You would have added those files in the root folder not in the .gitignore file.